A6.md

```diff
@@ -214,15 +217,15 @@ To clarify the second scenario, we define an *outgoing message* as everything th
 
 #### Buffered RPCs
 
-The gRPC library will have a configurable amount of available memory, `retry_buffer_size`, to buffer outgoing retryable or hedged RPCs. There is also a per-RPC size limit ,`per_rpc_buffer_limit`. These limits are configured by the client, rather than coming from the service config.
+The gRPC library will have a configurable amount of available memory, `retry_buffer_size_in_bytes`, to buffer outgoing retryable or hedged RPCs. There is also a per-RPC size limit ,`per_rpc_buffer_limit_in_bytes`. These limits are configured by the client, rather than coming from the service config.
```

Comment:
I suggest changing the wording to make it clear that this part of the spec is optional and may differ based on how the client is implemented.  Suggested text (feel free to wordsmith):

The gRPC client library may support application-configured limits for the amount of memory used for retries.  It is suggested that the client limit both the total amount of memory used to buffer retryable or hedged RPCs as well as the amount of memory used by any one RPC (to prevent a single large RPC from using the whole buffer and preventing retries of subsequent smaller RPCs).